packages/react/cypress/support/auto.tsx

```diff
@@ -3,16 +3,23 @@ import "@shopify/polaris/build/esm/styles.css";
 import translations from "@shopify/polaris/locales/en.json";
 import type { ComponentType, ReactNode } from "react";
 import React from "react";
+import { Toaster, elements } from "../../spec/auto/shadcn-defaults/index.js";
 import type { AutoAdapter } from "../../src/auto/index.js";
 import * as PolarisAdapter from "../../src/auto/polaris/index.js";
+import { makeAutocomponents } from "../../src/auto/shadcn/index.js";
 import { FormProvider, useForm } from "../../src/useActionForm.js";
+import { SUITE_NAMES } from "./constants.js";
 
 interface AutoSuiteConfig {
   name: string;
   adapter: AutoAdapter;
   wrapper: ComponentType<{ children: ReactNode }>;
 }
 
+const ONLY_RUN_SUITES = {
+  [SUITE_NAMES.SHADCN]: ["AutoButton"],
```

Comment:
I am down for this, but it does mean all the autoinputs stuff that is landing in this PR is untested for now, right?